<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kutuphane</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Bağlantı</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>160</width>
        <height>93</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QLineEdit" name="txt_sifre">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Kullanıcı Adı</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txt_kullanici_adi"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Şifre</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Durum</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lblDurum">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_kes">
         <property name="text">
          <string>Bağlantıyı kes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_baglan">
         <property name="text">
          <string>Bağlan</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="ogrKayit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Öğrenci Kayıt</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>261</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="soyadi_line"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="adi_line"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="soyadi">
         <property name="text">
          <string>Soyadı:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="adi">
         <property name="text">
          <string>Adı:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="sehir">
         <property name="text">
          <string>Şehir:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="sehir_box"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_kaydet">
         <property name="text">
          <string>Kayıt Ol</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="kitap_kayit">
     <attribute name="title">
      <string>Kitap Kayıt</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>29</y>
        <width>271</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="kitap_adi"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Adı:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Yazar:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="kitap_yazar"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="kayit_buton">
         <property name="text">
          <string>Kayıt</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="ogrenci_listele">
     <attribute name="title">
      <string>Öğrenci Listele</string>
     </attribute>
     <widget class="QPushButton" name="listele_btn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Listele</string>
      </property>
     </widget>
     <widget class="QListWidget" name="ogrenci_listesi">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txt_kullanici_adi</tabstop>
  <tabstop>txt_sifre</tabstop>
  <tabstop>btn_kes</tabstop>
  <tabstop>btn_baglan</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>adi_line</tabstop>
  <tabstop>soyadi_line</tabstop>
  <tabstop>sehir_box</tabstop>
  <tabstop>btn_kaydet</tabstop>
  <tabstop>kitap_adi</tabstop>
  <tabstop>kitap_yazar</tabstop>
  <tabstop>kayit_buton</tabstop>
  <tabstop>listele_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
